website/docs/r/user.html.markdown

```diff
@@ -64,6 +64,7 @@ The following arguments are supported:
 * `primary_group` - (Optional) Primary group ID of the User. Defaults to 0 (oneadmin).
 * `groups` - (Optional) List of secondary groups ID of the user.
 * `quotas` - (Optional) See [Quotas parameters](#quotas-parameters) below for details
+* `tags` - (Optional) Group tags (Key = value)
```

Comment:
Add an example in the HCL sample instead of this inline indication ('Key = value')